### PR TITLE
add version filter

### PR DESCRIPTION
**Pull request by mellisio**

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -22,6 +22,13 @@ class CondaAPI < Sinatra::Base
     Conda.instance.find_package(params["name"]).to_json
   end
 
+  get "/package/:name/:version" do
+    content_type :json
+    package = Conda.instance.find_package(params["name"])
+    package[:versions] = package[:versions]&.select { |version| version[:number] == params["version"] }
+    package.to_json
```
Comment (jsonperl) [inline app.rb:29]: Is it ok to send a null back if we don't find it?

Comment (jsonperl) [reply app.rb:29]: Hmm actually that'll explode.

Comment (mellisio) [reply app.rb:29]: send null back? It will 404 if we don't find the package

Comment (jsonperl) [reply app.rb:29]: Ah right! Cool.

**Review by jsonperl — APPROVED** [review]

**Pull request merged**